### ISSUE TITLE
Cannot create/import projects at all Eclipse Che 6.19

### ISSUE THREAD

**Issue by featherfeet** [eclipse-che/che#14789]
### Description:
Eclipse Che cannot create new projects or import any projects.
### Che version:
Eclipse Che 6.19.0 on Docker, running like this:
`docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -v /home/oliver/eclipse_che:/data eclipse/che:6.19.0 start`
### Steps to reproduce:
Run Che with the above command. Create a Java/MySQL workspace. Press "Create Project" or "Import Project."
### Expected Behavior:
Creating a project opens a dialog with no templates that greys out the "Create" button. Importing brings up a red error box saying "Failed to import project."
### Runtime:
```
docker on Debian 9
docker version:
Client: Docker Engine - Community
 Version:           19.03.2
 API version:       1.40
 Go version:        go1.12.8
 Git commit:        6a30dfca03
 Built:             Thu Aug 29 05:29:49 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.2
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.8
  Git commit:       6a30dfca03
  Built:            Thu Aug 29 05:28:23 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.6
  GitCommit:        894b81a4b802e4eb2a91d1ce216b8817763c29fb
 runc:
  Version:          1.0.0-rc8
  GitCommit:        425e105d5a03fabd737a126ad93d62a9eeede87f
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```
### Installation method:
I just ran `docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -v /home/oliver/eclipse_che:/data eclipse/che:6.19.0 start`
### Environment
Debian 9 (Stretch)

**Comment by tolusha** [comment]
@featherfeet 
Is there any particular reason you are trying to run 6.x version?

**Comment by tolusha** [comment]
It is better to try 7.x version.

Doc: https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che/
How to install locally: https://www.eclipse.org/che/docs/che-7/running-che-locally/
Blog: https://che.eclipse.org/eclipse-che-7-is-now-available-40ae07120b38

**Comment by g3ngar** [comment]
I think it's because `/projects` does not have write permissions for user `user`. I do not want to install kubernetes and I can't find docker instructions in the che 7 documentation, that's why I'm using che 6. Is any workaround for this issue?
